### PR TITLE
Feature/pxblue 2513 blui

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Go BLUI

### CODE REVIEW

File: src/app/drawer/drawer.component.spec.ts [hunk 1]
```diff
@@ -137,16 +137,16 @@ describe('DrawerComponent', () => {
     });
```
Comment (andrupu) [inline src/app/drawer/drawer.component.spec.ts:137]: Line 135, We might want to change 'pxb-drawer-collapse' to 'blui-drawer-collapse'.

File: src/app/drawer/drawer.component.ts [hunk 2]
```diff
@@ -9,43 +9,43 @@ import { NavigationEnd, Router } from '@angular/router';
...
                             (select)="nestedItem.onSelect(); setActive(navItem.title, nestedItem.title)"
-                        ></pxb-drawer-nav-item>
-                        <i pxb-icon *ngIf="navItem.icon === 'pxblue'" class="pxb-pxblue_small"></i>
-                        <mat-icon pxb-icon *ngIf="navItem.icon !== 'pxblue'">{{ navItem.icon }}</mat-icon>
-                    </pxb-drawer-nav-item>
-                </pxb-drawer-nav-group>
-            </pxb-drawer-body>
-            <pxb-drawer-footer>
+                        ></blui-drawer-nav-item>
+                        <i blui-icon *ngIf="navItem.icon === 'blui'" class="blui-pxblue_small"></i>
```
Comment (andrupu) [inline src/app/drawer/drawer.component.ts:33]: How do we want to get rid of "pxblue" in 'pxb-pxblue-small'?

File: src/app/drawer/drawer.component.spec.ts (hunk 1 above)
Comment (emclaug2) [reply src/app/drawer/drawer.component.spec.ts:117]: This unit test wasn't working correctly so I did some small unit test refactoring.  Should be removed now.

File: src/app/drawer/drawer.component.ts (hunk 2 above)
Comment (emclaug2) [reply src/app/drawer/drawer.component.ts:33]: The icon is actually called `pxblue_small` so we can't remove this pxb ref.  The prefix was updated to be `blui` though.

Comment (andrupu) [reply src/app/drawer/drawer.component.ts:33]: Okay, I didn't know if we wanted to change the name of pxblue_small icon to be more blui.  The prefix looks great.

**Review by andrupu — APPROVED** [review]

**Pull request merged**